src/ui/studio/save-creation/index.js

```diff
@@ -287,12 +289,29 @@ const ConnectionUnconfiguredWarning = () => {
 
 const UploadForm = ({ opencast, uploadState, recordings, handleUpload }) => {
   const { t } = useTranslation();
+  let series = opencast.getFinalSeriesTitles();
+  let titles = [];
+  for (let keys of series.keys()){
+    let tmp = {value:keys,label:series.get(keys)};
+    titles.push(tmp);
+  }
+
+
+  const enabledSeries = useSettings().upload?.enableSeries;
+  const displaySeries =  enabledSeries  === 'true' ? 'block' : 'none';
+  const enableDescription = useSettings().upload?.enableDescription;
+  const displayDescription = enableDescription === 'true' ? 'block' : 'none';
```

Comment:
You should call `useSettings()` once and store the result as `settings`. As far as I know, it's not too great to call these react hooks more than necessary.